src/crush/CrushLocation.cc

```diff
@@ -39,7 +39,6 @@ int CrushLocation::_parse(const std::string& s)
   }
   std::lock_guard l(lock);
   loc.swap(new_crush_location);
-  lgeneric_dout(cct, 10) << "crush_location is " << loc << dendl;
```

Comment:
i think, instead of removing this logging message, we should remove the one in `CrushLocation::init_on_startup()`. because `CephContext::CephContext()` sets `_log` *after* constructing `crush_location`, the value of `_log` is random bits on heap, not to mention the member variables of `*_log`.

also, could you create a tracker ticket at CephContext , and revise the commit message? like

> crush/CrushLocation: do not print logging message in constructor
>
> do not use logging facility in constructor, as `CephContext::_log` is set after `CephContext::crush_location` is created in constructor of `CephContext`.
>
> Fixes: https://tracker.ceph.com/issues/<tracker-id>

Reply:
yes, you are right, I should remove the log message at CrushLocation::init_on_startup().
It has errors where I register account at https://tracker.ceph.com/account/register  
<img width="375" alt="微信图片_20210329164819" src="https://user-images.githubusercontent.com/1087071/112811401-9eed6780-90ae-11eb-8fc7-c106e227aeb7.png">
My email is notmycupoftea@163.com, thanks!

Reply:
@lihuibng i cannot do this for you. if you have trouble registering an account. i can create a tracker ticket on behalf of you.

Reply:
Thank you for creating a tracker ticket for me!

Reply:
@tchaikov  I have add the "Signed Off By", and commit the fix again.